Dockerfile

```diff
@@ -14,32 +14,91 @@ FROM build-setup AS build-dependencies
 
 # Cache gopath dependencies for faster builds
 # Newer projects should opt for go mod vendor for reliability and security
-WORKDIR /app/src
+RUN mkdir /app
+RUN mkdir /app/src
 COPY src/go.mod /app/src
 COPY src/go.sum /app/src
-RUN go mod download
 
 # FIX: This generates code marked by `go:build relic` and `+build relic`. See `combined_verifier_v3.go`.
+# FIX: This is not needed, if vendor/ is used
+WORKDIR /app/src
+RUN go mod download
 RUN go mod download github.com/onflow/flow-go/crypto@v0.24.3
 RUN cd $GOPATH/pkg/mod/github.com/onflow/flow-go/crypto@v0.24.3 && go generate && go build
 
+# FIX: Devs should review all what they use to limit build time
+RUN cat go.sum

```

Comment:
is this step necessary?

Reply:
It is not essential. It is a good debugging measure due to the external copy or if somebody adds back the tidy step
```
COPY src/go.mod /app/src
COPY src/go.sum /app/src
```

Reply:
Removed, thanks